release/obd-kardinal.yaml

```diff
@@ -129,14 +129,22 @@ spec:
                 - name: "Cookie"
                   value: "shop_session-id=x-liveness-probe"
           env:
-            - name: ADDRESS
-              value: ":8080"
             - name: JSDELIVRAPIKEY
               value: "prod"
             - name: CARTSERVICEHOST
               value: cartservice
             - name: PRODUCTCATALOGSERVICEHOST
               value: productcatalogservice
+            #- name: AWS_REGION
```

Comment:
commented on these because this functionality will be available only if these values are set, and the default configuration is not to pass them at least you want to test this use case.
The code handles either case